mavros/src/plugins/setpoint_raw.cpp

```diff
@@ -179,19 +197,25 @@ class SetpointRawPlugin : public plugin::PluginBase,
 		// Transform frame ENU->NED
 		velocity = ftf::transform_frame_enu_ned(velocity);
 		af = ftf::transform_frame_enu_ned(af);
-		yaw = ftf::transform_frame_yaw_enu_ned(req->yaw);
-		yaw_rate = ftf::transform_frame_yaw_enu_ned(req->yaw_rate);
+		yaw = ftf::quaternion_get_yaw(
+					ftf::transform_orientation_aircraft_baselink(
+						ftf::transform_orientation_ned_enu(
+							ftf::quaternion_from_rpy(0.0, 0.0, req->yaw))));
+		Eigen::Vector3d ang_vel_enu(0.0, 0.0, req->yaw_rate);
+		auto ang_vel_ned = ftf::transform_frame_ned_enu(
+					ftf::transform_frame_aircraft_baselink(ang_vel_enu));
+		yaw_rate = ang_vel_ned.z();
```

Comment:
`yaw_rate = - req->yaw_rate;`

Reply:
Remove `fft::transform_orientation_aircraft_baselink()` part and try again.